Source/WebCore/platform/graphics/cocoa/IOSurface.h

```diff
@@ -36,6 +36,7 @@
 #if PLATFORM(IOS_FAMILY) && !PLATFORM(MACCATALYST) && !PLATFORM(IOS_FAMILY_SIMULATOR)
 #define HAVE_IOSURFACE_RGB10 1
 #endif
+#define HAVE_IOSURFACE_RGB10 1
```

Comment:
This change is probably accidental (and unwanted). Can you revert this? It's also breaking your build.